service/src/test/java/uk/gov/hmcts/reform/fpl/utils/NotifyAdminOrderIssuedTestHelper.java

```diff
@@ -0,0 +1,119 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.MapDifference;
+import com.google.common.collect.Maps;
+import org.apache.commons.codec.binary.Base64;
+import org.json.JSONObject;
+import org.mockito.ArgumentCaptor;
+import uk.gov.hmcts.reform.fpl.enums.IssuedOrderType;
+import uk.gov.hmcts.reform.fpl.model.Address;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.nio.charset.StandardCharsets.ISO_8859_1;
+import static org.assertj.core.api.Assertions.assertThat;
+import static uk.gov.hmcts.reform.fpl.enums.IssuedOrderType.CMO;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.POST;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.wrapElements;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.formatCaseUrl;
+
+public class NotifyAdminOrderIssuedTestHelper {
+
+    private NotifyAdminOrderIssuedTestHelper() {
+    }
```

Comment:
is constructor required?

Reply:
nope 👍 you're right

Reply:
```
[ant:checkstyle] [WARN] reform/fpl/utils/NotifyAdminOrderIssuedTestHelper.java:25:1: Utility classes should not have a public or default constructor. [HideUtilityClassConstructor]
```
nevermind, it's required

Reply:
lombok annotation makes public constructors, need private one here